### PR TITLE
Update regarding button subscription changes for Resumption error handling scripts

### PR DESCRIPTION
**Pull request by GetmanetsIrina**
Internal review

This PR is **ready** for review.

### Summary
Update regarding button subscription changes for Resumption error handling scripts

### ATF version
https://github.com/smartdevicelink/sdl_atf/tree/develop

### CODE REVIEW

**Comment by GetmanetsIrina** [comment]
Please review only last commit https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2125/commits/ece176a6891d7fa2274e5b269ae3ace37b5d3479

File: test_scripts/Resumption/Handling_errors_from_HMI/020_resume_all_data_without_response_to_one_rpc_2apps_ign_off.lua [hunk 1]
```diff
@@ -77,7 +78,7 @@ for k, value in pairs(rpcs) do
     runner.Step("openRPCserviceForApp1", common.openRPCservice, { 1 })
     runner.Step("openRPCserviceForApp2", common.openRPCservice, { 2 })
     runner.Step("Reregister Apps resumption error to " .. interface .. " " .. k,common.reRegisterApps,
-      {common.checkResumptionData2Apps, k, interface, 12000})
+      {common.checkResumptionData2Apps, k, interface, 13000})
```
[the remark this answers is not in the document]
Comment (GetmanetsIrina) [reply test_scripts/Resumption/Handling_errors_from_HMI/020_resume_all_data_without_response_to_one_rpc_2apps_ign_off.lua]: This change is related to cases where there is no HMI response with 2 apps , updated also in 019.
Changes in common is related to request receiving, 12000 was not enough to receive revert requests with 2 apps.

File: test_scripts/Resumption/Handling_errors_from_HMI/commonResumptionErrorHandling.lua [hunk 2]
```diff
@@ -792,6 +829,26 @@ function m.subscribeWayPointsResumption(pAppId, pErrorResponseInterface)
     end)
 end
 
+--[[ @subscribeButtonResumption: check resumption of subscribeWayPoints data
```
[the remark this answers is not in the document]
Comment (GetmanetsIrina) [reply test_scripts/Resumption/Handling_errors_from_HMI/commonResumptionErrorHandling.lua]: Updated.

File: test_scripts/Resumption/Handling_errors_from_HMI/020_resume_all_data_without_response_to_one_rpc_2apps_ign_off.lua (hunk 1 above)
[the remark this answers is not in the document]
Comment (dboltovskyi) [reply test_scripts/Resumption/Handling_errors_from_HMI/020_resume_all_data_without_response_to_one_rpc_2apps_ign_off.lua]: @GetmanetsIrina Agreed

File: test_scripts/Resumption/Handling_errors_from_HMI/commonResumptionErrorHandling.lua [hunk 3]
```diff
@@ -342,6 +351,19 @@ m.rpcsRevert = {
         end)
       :Times(pTimes)
     end
+  },
+  subscribeButton = {
+    Buttons = function(pAppId,pTimes)
+      if not pTimes then pTimes = 1 end
+      m.getHMIConnection():ExpectRequest("Buttons.SubscribeButton",
+        { buttonName = "CUSTOM_BUTTON", appID = m.getHMIAppId(pAppId) },
```
[the remark this answers is not in the document]
Comment (dboltovskyi) [reply test_scripts/Resumption/Handling_errors_from_HMI/commonResumptionErrorHandling.lua]: @GetmanetsIrina Agreed

**Pull request merged**